### PR TITLE
Run JSC tests using jsvu's JSC

### PR DESCRIPTION
**Pull request by dschuff**
Also switch to testing the lld version of the bare tests, since that's the way forward.

### CODE REVIEW

**Comment by dschuff** [comment]
All the tests are passing already!
/cc @mathiasbynens

**Comment by mathiasbynens** [comment]
This is awesome! I love the simplification :)

**Comment by mathiasbynens** [comment]
This is awesome! I love the simplification of no longer having to clone the entire 9 GB+ WebKit repo and building JavaScriptCore from source 👍

File: src/build.py [hunk 1]
```diff
@@ -84,7 +83,7 @@
 
 LLVM_OUT_DIR = os.path.join(WORK_DIR, 'llvm-out')
 V8_OUT_DIR = os.path.join(V8_SRC_DIR, 'out.gn', 'x64.release')
-JSC_OUT_DIR = os.path.join(JSC_SRC_DIR, 'current-release')
+JSC_OUT_DIR = os.path.expanduser(os.path.join('~', '.jsvu'))
```
Comment (jgravelle-google) [inline src/build.py]: TIL about `expanduser`, nice

**Comment by dschuff** [comment]
Going ahead and landing with a TODO since it's strictly an improvement over the current (usually-broken) state of testing on JSC.

File: src/build.py [hunk 2]
```diff
@@ -1002,41 +997,10 @@ def Jsvu():
...
 
-  except:
-    buildbot.Warn()
-
-
-def Jsc():
-  buildbot.Step('JSC')
-  Mkdir(JSC_OUT_DIR)
+    # TODO: Install the JSC binary in the output package, and add the version
+    # info to the repo info JSON file (currently in GetRepoInfo)
```
Comment (mathiasbynens) [inline src/build.py]: FWIW, `~/.jsvu/status.json` contains the installed version number.

**Pull request merged**